### PR TITLE
fix: martial arts damage buffs apply to all damage types

### PR DESCRIPTION
**Pull request by yay855**
## Purpose of change

There are multiple unarmed martial arts weapons ingame that perform a variety of damage types, but many martial arts techniques and buffs only buffed bashing damage.

## Describe the solution

All martial arts techniques and buffs that only buffed one or two of the three main melee damage types (bash, cut, stab) now buff all three.

### CODE REVIEW

File: data/json/martialarts.json [hunk 1]
```diff
@@ -157,7 +157,14 @@
...
         "description": "A solid stance allows you block more damage than normal and deliver better punches.\n\n+2 Bash damage, Blocked damge reduced by 50% of Strength.",
         "unarmed_allowed": true,
-        "flat_bonuses": [ { "stat": "block", "scaling-stat": "str", "scale": 0.5 }, { "stat": "damage", "type": "bash", "scale": 2.0 } ]
+        "flat_bonuses": [
+          { "stat": "block", "scaling-stat": "str", "scale": 0.5 },
+          { "stat": "damage", "type": "bash", "scale": 2.0 },
+          { "stat": "block", "scaling-stat": "str", "scale": 0.5 },
+          { "stat": "damage", "type": "cut", "scale": 2.0 },
+          { "stat": "block", "scaling-stat": "str", "scale": 0.5 },
+          { "stat": "damage", "type": "stab", "scale": 2.0 }
```
Comment (scarf005) [inline data/json/martialarts.json:166]: identical `{ "stat": "block", "scaling-stat": "str", "scale": 0.5 }` bonus is repeated three times, would this be intentional?

Comment (chaosvolt) [inline data/json/martialarts.json:166]:
```suggestion
          { "stat": "block", "scaling-stat": "str", "scale": 0.5 },
          { "stat": "damage", "type": "bash", "scale": 2.0 },
          { "stat": "damage", "type": "cut", "scale": 2.0 },
          { "stat": "damage", "type": "stab", "scale": 2.0 }
```
Definitely just a copypaste error it looks like.

**Review by chaosvolt — COMMENTED** [review]
Rest of it looks good, though I'd note that most of these buffs specify what damage types they modify, so should edit those to not reference a specific damage type.

File: data/json/martialarts.json (hunk 1 above)
Comment (yay855) [reply data/json/martialarts.json]: Oh, it wouldn't. I must have accidentally copied it alongside the damage scaling. I'll fix it.

Comment (yay855) [reply data/json/martialarts.json]: Oh, Chaos already did.

**Review by scarf005 — CHANGES_REQUESTED** [review]
like CV said, I think we should change all descriptions to reflect PR, e.g `+2 Bash damage` to `+2 damage` to correctly describe changes.

**Review by scarf005 — APPROVED** [review]
LGTM

**Pull request merged**